about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID Enterprise Geodatabase.
 
-## Adding New Tables
+## Adding New Datasets to the SGID
 {: .text-left }
 
-When adding new data to the SGID, the steps outlined in the [Porter](https://github.com/agrc/porter) project will be followed.
+We strive to achieve and maintain a high standard of quality for the SGID. One of the best ways to do this is making sure all datasets coming into the SGID are high-quality datasets. We use the following policies and procedures to guide this effort.
 
-Additional, related info:
+### Qualifying Datasets
+{: .text-left }
+
+Datasets requested to be added to the SGID by a state or local agency or organization should meet all the following criteria:
+
+- The dataset should cover a **state-wide** geographic extent, be part of a state-wide project or initiative (such as roads, address points, and parcels), or be relevant to broader projects or mandates that have a state-wide interest (hazard mapping or endangered species mapping).
+- The dataset should be useful to the daily activities of **multiple agencies** or organizations. Datasets primarily used internally by a single agency don't belong in the SGID.
+- The entity providing the dataset should be the **authoritative** or original source (for example, a county's parcels must come from the County government).
+- The entity providing the dataset does not have the **ability** to publicly share data (such as an existing ArcGIS Online organization). An entity's data storage costs or server capacity may not be sufficient justification for adding data to the SGID.
```

Comment:
```suggestion
- The entity providing the dataset does not have the **ability** to publicly host the data within their organization (such as an existing ArcGIS Online organization). An entity's data storage costs or server capacity may not be sufficient justification for adding data to the SGID.
```

Reply:
I like this bullet point a lot (it encourages the new model where you host we catalog).  but, again, i think this might be confusing unless we define what the sgid is.  hopefully, the first few sentences of this page can define that.  b/c we are saying here we will bring it into Internal/OpenData/OpenSGID if you can't host your data, but if you can host it, and the other criteria are met, then it can be added to OpenData but you host it.  it's tricky.

Reply:
Your suggestion to explain what the SGID is now will be driven home by this if we mention that it is distributed and we are the index or something similar.

Reply:
I like the distinction between host and share, but I updated the wording to avoid using organization repetitively. Let me know what you think.

Reply:
I think i like share more than host since sharing includes agol and portal while hosting might imply not using the cloud?

Reply:
`does not have the internal **capability** to publicly serve the data (such as through an existing ArcGIS Online organization)`?

Reply:
nice!  like it.  ...share or host both work for me.

Reply:
ship it!